xdsl/tools/xdsl_tool.py

```diff
@@ -0,0 +1,218 @@
+import argparse
+import os
+import sys
+from typing import IO, Callable
+
+from xdsl.backend.riscv.lowering.riscv_arith_lowering import RISCVLowerArith
+from xdsl.dialects.affine import Affine
+from xdsl.dialects.arith import Arith
+from xdsl.dialects.builtin import Builtin, ModuleOp
+from xdsl.dialects.cf import Cf
+from xdsl.dialects.cmath import CMath
+from xdsl.dialects.experimental.dmp import DMP
+from xdsl.dialects.experimental.fir import FIR
+from xdsl.dialects.experimental.math import Math
+from xdsl.dialects.func import Func
+from xdsl.dialects.gpu import GPU
+from xdsl.dialects.irdl.irdl import IRDL
+from xdsl.dialects.linalg import Linalg
+from xdsl.dialects.llvm import LLVM
+from xdsl.dialects.memref import MemRef
+from xdsl.dialects.mpi import MPI
+from xdsl.dialects.pdl import PDL
+from xdsl.dialects.printf import Printf
+from xdsl.dialects.riscv import RISCV
+from xdsl.dialects.riscv_func import RISCV_Func
+from xdsl.dialects.scf import Scf
+from xdsl.dialects.snitch import Snitch
+from xdsl.dialects.snitch_runtime import SnitchRuntime
+from xdsl.dialects.stencil import Stencil
+from xdsl.dialects.test import Test
+from xdsl.dialects.vector import Vector
```

Comment:
Could we import this like `from xdsl.dialects import affine, arith, ...` and then have `affine.Affine` below? That would be more consistent with the way we want dialect import to look like in the repo.

(I know this is irrelevant for this PR, but since we touch the code anyhow....)

Reply:
How about we have something like `from xdsl.dialects import Arith, Builtin, ...` ? (In another PR though)
Otherwise, I think this current version makes more sense. `affine.Affine` seems like you have an `Affine` operation in the `affine` dialect.

Reply:
Yeah, that'd be even better :+1: